lua/entities/gmod_tardis_interior/modules/cl_customlighting_prompt.lua

```diff
@@ -0,0 +1,29 @@
+--Require Lighting Override
+
+TARDIS:AddSetting({
+    id="light_override_request_noshow",
+    name="don't show again",
+    value=false,
+    type="bool",
+})
+
+ENT:AddHook("PlayerInitialize", "customlightenable", function(self)
+    local require_override = (self.metadata.Interior.RequireLightOverride == true) 
+    local light_override = TARDIS:GetSetting("lightoverride-enabled", false)
+    local noshow = TARDIS:GetSetting("light_override_request_noshow", false)
+    if require_override and not light_override and not noshow then
+        Derma_Query(
+            "It is recommended to use Lighting Override with this TARDIS! This can be changed in the TARDIS settings later. Enable now?",
```

Comment:
The exclamation mark is a little jarring in this message, how about this?

```suggestion
            "This TARDIS recommends Lighting Override which you do not have enabled. Enable now? This can be changed in TARDIS settings later.",
```